src/replicache.ts

```diff
@@ -808,6 +813,38 @@ export class Replicache<MD extends MutatorDefs = {}> {
     this._pullConnectionLoop.send();
   }
 
+  /**
+   * Applies an update from the server to Replicache.
+   * Throws an error if cookie does not match. Host should re-init.
+   *
+   * @experimental - This method is under development and its semantics will change.
+   */
+  async poke(poke: Poke): Promise<void> {
+    await this._ready;
+    const clientID = await this._clientIDPromise;
+    const requestID = sync.newRequestID(clientID);
```

Comment:
I feel like this doesn't make much sense anymore unless it was set by the server. The point of it was to be able to tie actions together across client and server, so as it is it's a little bit useless.

Reply:
Agree.